extensions/amp-story/1.0/page-advancement.js

```diff
@@ -560,6 +571,30 @@ export class ManualAdvancement extends AdvancementConfig {
     );
   }
 
+  /**
+   * Checks if click target is too large on the page and preventing navigation.
+   * If yes, the link is ignored & logged.
+   * @param {!Element} target
+   * @param {!ClientRect} pageRect
+   * @return {boolean}
+   * @private
+   */
+  isTooLargeOnPage_(target, pageRect) {
+    const targetRect = target./*OK*/ getBoundingClientRect();
+    if (
+      (targetRect.height * targetRect.width) /
+        (pageRect.width * pageRect.height) >=
+      MAX_LINK_SCREEN_PERCENT
+    ) {
+      user().error(
+        'AMP-STORY-PAGE',
+        'Link occupies over 80% of page size; overriden to allow for navigation.'
```

Comment:
nit: maybe we can just say something along the lines of "link was too large. Skipped for navigation" and include a link to our docs?

The 80% number here seems like it could change over time, and it would be great to consolidate our guidance on how to be compliant

Reply:
Changed this to: `Link was too large; skipped for navigation. For more information, see https://amp.dev/documentation/components/amp-story/`

Reply:
Actually, I added some documentation to amp-story.md and linked to that in this error.